nemo_rl/environments/games/sliding_puzzle.py

```diff
@@ -350,13 +350,13 @@ def __init__(self, cfg: Optional[SlidingPuzzleConfig] = None):
     def step(
         self,
         message_log_batch: list[LLMMessageLogType],
-        metadata_batch: list[SlidingPuzzleMetadata],
-    ) -> EnvironmentReturn:
+        metadata: list[SlidingPuzzleMetadata],
```

Comment:
what allows for this function to use 'list' in the type when we have to use 'Sequence' elsewhere?